nixos/modules/services/games/factorio.nix

```diff
@@ -120,6 +201,7 @@ in
           "--config=${cfg.configFile}"
           "--port=${toString cfg.port}"
           "--start-server=${mkSavePath cfg.saveName}"
+          "--server-settings=${serverSettingsFile}"
           (optionalString (cfg.mods != []) "--mod-directory=${modDir}")
           (optionalString (cfg.autosave-interval != null) "--autosave-interval ${toString cfg.autosave-interval}")
```

Comment:
I didn't see an option in the [sample file](https://gist.github.com/bflyblue/d10380e724936e152ce63ae93c324ea6) bundled with Factorio to specify the mods so I only exposed a few options I thought may be useful.